src/test/java/org/mongojack/TestDBUpdateSerialization.java

```diff
@@ -153,6 +157,30 @@ public void testSimpleMap() {
         assertThat(coll.findOneById("id").map, equalTo(map));
     }
 
+    // Test to detect presence of issue https://github.com/devbliss/mongojack/issues/98
+    @Test
+    public void testUpdateOfNestedRepeatedAttributeName() {
+        coll2 = getCollection(NestedRepeatedAttributeName.class, String.class);
+
+        Calendar cal = Calendar.getInstance();
+        cal.set(0, 0, 0, 0, 0, 10);
+        Date d_1 = cal.getTime();
```

Comment:
`_` is kind of uncommon in Java naming. I'd probably just call them `d1` and `d2`